src/features/devtools/DevTools.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from 'react';
+import type { ReactNode } from 'react';
+
+import { OpenDevToolsButton } from 'src/features/devtools/components/OpenDevToolsButton/OpenDevToolsButton';
+import { DevToolsPanel } from 'src/features/devtools/DevToolsPanel';
+
+const devToolsEnabled = window.location.hostname === 'local.altinn.cloud';
+
+interface IDevToolsProps {
+  children: ReactNode;
+}
+
+export const DevTools = ({ children }: IDevToolsProps) => {
+  const [panelOpen, setPanelOpen] = useState(false);
+
+  useEffect(() => {
+    if (devToolsEnabled) {
+      return window.addEventListener('keydown', (event) => {
+        if ((event.ctrlKey || event.metaKey) && event.key === 'k') {
```

Comment:
Isn't this shortcut rather short and likely for normal users to press inadvertently.

Reply:
I will gladly take suggestions for other shortcuts, maybe just add `Shift`?

Reply:
I think i missunderstood this. It's currently just a shortcut for when the icon is already visible?

I'm thinking about when this is activated in prod, the mode should probably be activated by something else than a key combination that others might have configured for other purposes (or inadvertently type). Personally I'd suggest setting a custom cookie (devtoolsEnable:true) from javascript when the user adds `?devtools=` in their url. The devtools can provide a button to delete the cookie, and docs entries can link to any app to help users set the cookie. 

PS: I don't think code splitting should be dependent on `Index.cshtml`, but I have not tried setting it up myself. I just used Create React App.

Reply:
I believe we'll actually want to have the devtools somewhat easily accessible in prod as well. The goal here is not to hide anything, even from competent users (they could figure all of this out without devtools if they wanted), and using a cookie or custom URL just adds more friction than we really need. My goal is to make these tools available on mobile as well some day, and making mobile users inject a cookie will make it very hard for them to use the tools. (Keyboard shortcuts isn't all too easy on mobile either, but we'll figure something out in time.)